packages/rollup-core/src/app/data/producers/l2-chain-data-persister.ts

```diff
@@ -35,18 +31,21 @@ export class L2ChainDataPersister extends ChainDataProcessor {
    * @param db The DB to use to persist the queue of Block objects.
    * @param dataService The L2 Data Service handling persistence of relevant data.
    * @param l2Provider The provider to use to connect to L2 to subscribe & fetch block / tx data.
+   * @param earliestBlock The earliest block to sync.
```

Comment:
```suggestion
   * @param earliestBlock The earliest L1 block to sync from.
```

Reply:
This should actually be for L2 though. I may just leave these as the context is implied by the file they're in.